src/kernel/include/shared/cos_config.h

```diff
@@ -17,23 +17,19 @@
 
 #include "cpu_ghz.h"
 
-/* we already use 16 MB of kernel memory + the kernel img, thus 32MB offset for this: */
-#define COS_MEM_USER_PA       (1<<25)
-#define COS_MEM_USER_PA_SZ    (1<<25) /* start with 32MB of memory */
 
 /*
  * 1 MB, note that this is not the PA of kernel-usable memory, instead
  * it is the PA of the kernel.  If you change this, update the kernel
  * linker script (.ld) as well.
  */
 #define COS_MEM_KERN_PA (0x00100000)
-#define COS_MEM_KERN_PA_ORDER (25)
+#define COS_MEM_KERN_PA_ORDER (29)
```

Comment:
There weren't any. I checked through the code for its usage and also executed micro_booter and vkernel. This has also been in effect in the rump2cos for a while now and haven't had any issues with this.